alpine/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM alpine:latest
+MAINTAINER Peter Martini <PeterCMartini@GMail.com>
+
+RUN mkdir -p /usr/src/perl
```

Comment:
As of a semi-recent Docker version, this bit is no longer necessary (`COPY` and `WORKDIR` will both properly create the directory as necessary).